pydra/engine/specs.py

```diff
@@ -444,7 +464,16 @@ def collect_additional_outputs(self, inputs, output_dir):
         additional_out = {}
         for fld in attr_fields(self):
             if fld.name not in ["return_code", "stdout", "stderr"]:
-                if fld.type in [File, MultiOutputFile]:
+                if fld.type in [
+                    File,
+                    MultiOutputFile,
+                    Directory,
+                    Int,
+                    Float,
+                    Bool,
+                    Str,
+                    List,
```

Comment:
i think you should be able to use python types for this. i don't think you need special classes. did you try the Python types?

Reply:
Good point - I was following the existing pattern in which a class for File was defined, but I do think it would be better to use Python types. I will make the change and update the PR